src/main/kotlin/snyk/common/CustomEndpoints.kt

```diff
@@ -100,8 +100,7 @@ internal fun resolveCustomEndpoint(endpointUrl: String?): String {
 
 fun URI.isSnykTenant() =
     isSnykDomain()
-        && ((host.lowercase().startsWith("app.") && path.lowercase().endsWith("/api"))
-        || (host.lowercase() == "snyk.io" && path.lowercase().endsWith("/api"))
+        && ((host.lowercase() == "snyk.io" && path.lowercase().endsWith("/api"))
```

Comment:
The ticket mentions that we should not use `app.snyk.io/api` but nothing about `snyk.io/api`. This is a legacy way of calling APIs, as far as I know, which works today with `api.snyk.io` too. I think we could improve the validation some more so `snyk.io/api` is not allowed but it's not something we were asked to do and I don't know for sure that this wouldn't break something else so I'd rather leave it out